verilog/analysis/checkers/no_trailing_spaces_rule.cc

```diff
@@ -59,7 +59,7 @@ void NoTrailingSpacesRule::HandleLine(absl::string_view line) {
   if (rbegin != rend) {
     // Lines already exclude \n, so we can check using std::isspace.
```

Comment:
this comment is now outdated.

Reply:
The comments have been updated so that they match what actually happens in the code.